lib/src/components/NxAccordion/__tests__/NxAccordion.test.tsx

```diff
@@ -5,313 +5,202 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { mount } from 'enzyme';
-import { faChevronCircleDown, faChevronCircleUp } from '@fortawesome/free-solid-svg-icons';
-import { act } from 'react-dom/test-utils';
+import userEvent from '@testing-library/user-event';
 
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
-import NxAccordion, { Props } from '../NxAccordion';
-import NxButton from '../../NxButton/NxButton';
-import NxFontAwesomeIcon from '../../NxFontAwesomeIcon/NxFontAwesomeIcon';
+import { rtlRender, rtlRenderElement } from '../../../__testutils__/rtlUtils';
 
-function createClickEvent() {
-  return new MouseEvent('click', { bubbles: true });
-}
+import NxAccordion from '../NxAccordion';
+import NxButton from '../../NxButton/NxButton';
+import { render } from '@testing-library/react';
 
 describe('NxAccordion', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent<Props>(NxAccordion, {}),
-      getMountedComponent = enzymeUtils.getMountedComponent<Props>(NxAccordion, {});
+  const quickRender = rtlRender(NxAccordion, {});
+  const renderEl = rtlRenderElement(NxAccordion, {});
 
   it('renders a <details> element with the provided props', function() {
-    const component = getShallowComponent({ id: 'foo', open: true, title: 'test title' });
-
-    expect(component).toMatchSelector('details#foo');
-    expect(component).toHaveProp('open', true);
-    expect(component).toHaveProp('title', 'test title');
+    const el = renderEl({ id: 'foo', open: true })!;
+    expect(el.tagName).toBe('DETAILS');
+    expect(el).toHaveAttribute('id', 'foo');
+    expect(el).toHaveAttribute('open');
```

Comment:
Since the `open` attr is especially crucial, I think we should have a separate test testing that it is set correctly for all possible values of the corresponding prop (`true`, `false`, and `undefined`).

Reply:
> Since the `open` attr is especially crucial, I think we should have a separate test testing that it is set correctly for all possible values of the corresponding prop (`true`, `false`, and `undefined`).

Added the test: e2508b1fddf6f6ae84c2ee5bb5ae1eecf2a4cd60